board/templates/home.html

```diff
@@ -1,472 +1,1016 @@
+      var hostname = window.location.origin
+      var user_root_url  = hostname + "/board/user"
+      var workspace_root_url = user_root_url + "/workspace"
+      var dataset_root_url = user_root_url + "/dataset"
+      debug = true
+      var userDatasets = [];
+      var availableRecommdenders = [];
+      var iframeTimeout = 10000
+
+      function getWorkspaceList(){
+          url = workspace_root_url + "/list"
+          if (debug){
+              console.log("listing workspaces",url)   
+          }
+          $.get( url, function( workspaces ) {
+              // console.log(workspaces)
+              addWorkspacesToUI(workspaces)
+          });
+      }
+
+      function getWorkspaceModelsList(wid,cb){
+          url =workspace_root_url + "/model/list"+"?wid="+wid
+          
+          if (debug){
+              console.log("listing workspace models",url) 
+          }
+
+          $.get(url, function( data ){
+                  // console.log("data:",data)
+                  data = eval(data)
+                  console.log("Listed workspace models:",data)
+                  if(cb && typeof cb === "function") {
+                      return cb(data);
+                  }
+          });
+      }
+
+      function getDatasetList(cb){
+          url = dataset_root_url + "/list"
+          if (debug){
+              console.log("listing datasets",url)   
+          }
+          $.get( url, function( data ) {
+              addDatasetsToUI(data)                    
+              $.each(data,function( _ , dataset){
+                  userDatasets.push(dataset)
+              });
+              $("#train-dataset-select").append(modelDatasetOptionsUI())
+              $("#test-dataset-select").append(modelDatasetOptionsUI())
+              if(cb && typeof cb === "function") {
+                  cb();
+              }
+          });
+      }
+
+
+      function getAvailableRecommenders(cb){
+          url = workspace_root_url + "/recommender/list"
+          if (debug){
+              console.log("listing available recommenders",url)   
+          }
+          $.get( url, function( data ) {
+              $.each(data,function( _ , dataset){
+                  availableRecommdenders.push(dataset)
+              });
+              $("#recommender-select").append(modelRecommenderOptionsUI());
+              if(cb && typeof cb === "function") {
+                  cb();
+              }
+          });
+      }
+
+      function modelDatasetOptionsUI(){
+          options = ""
+          $.each(userDatasets, function(i,ds){
+              options += '<option did="'+ds[0]+'">'+ds[1]+'</option>'
+          });
+          return options
+      }
+
+      function modelRecommenderOptionsUI(){
+          options = ""
+          $.each(availableRecommdenders, function(i,recommender){
+              options += "<option>"+recommender+"</option>"
+          });
+          return options  
+      }
+
+      // function createModelForm(workspace){
```

Comment:
Do we need this commented code?

Reply:
yes, same as before, model related migrations are not complete, i need this for reference.